nipype/info.py

```diff
@@ -148,6 +148,7 @@ def get_nipype_gitversion():
     'pydotplus',
     'pydot>=%s' % PYDOT_MIN_VERSION,
     'packaging',
+    'yapf'
```

Comment:
`yapf` is not a runtime dependency. it has been added to a new label called specs below.

Reply:
ah didn't see this earlier - sorry for the clutter